packages/shared-ux/page/analytics_no_data/src/index.ts

```diff
@@ -6,22 +6,5 @@
  * Side Public License, v 1.
  */
 
-import React from 'react';
-import { withSuspense } from '@kbn/shared-ux-utility';
-
 export { AnalyticsNoDataPageProvider, AnalyticsNoDataPageKibanaProvider } from './services';
-
-/**
- * Lazy-loaded connected component.  Must be wrapped in `React.Suspense`.
- */
-export const LazyAnalyticsNoDataPage = React.lazy(() =>
-  import('./analytics_no_data_page').then(({ AnalyticsNoDataPage }) => ({
-    default: AnalyticsNoDataPage,
-  }))
-);
-
-/**
- * An entire page that can be displayed when Kibana "has no data", specifically for Analytics.
- * Requires a Provider for relevant services.
- */
-export const AnalyticsNoDataPage = withSuspense(LazyAnalyticsNoDataPage);
+export { AnalyticsNoDataPage } from './analytics_no_data_page';
```

Comment:
I am confused - this component is a standalone component, so how come it doesn't need lazy loading?

Reply:
In this case, I felt like since the component was an entire _page_, there's not much benefit in lazy-loading it-- the solution is going to need it immediately on render anyway.